### DISCUSSION TITLE
Fork heroes 1 from fheroes2 codebase.

### DISCUSSION THREAD

**Discussion by DMJC** [ihhub/fheroes2#2640] [Ideas]
Heroes of Might an Magic 1 is basically a cut down version of Heroes 2. I propose forking the codebase and removing all the necromancer/wizard/elemental information from the codebase and then implementing any missing graphics changes to make Heroes 1 work. Is anyone else interested in working on this?

**Comment by ihhub** [comment]
Hi @DMJC ,

I don't think that this is a good idea. First of all, the games are different and the diffference is not only in graphics but in logic, mechanics and file processing techniques. Secondly, the fanbase for HoMM1 is extremely small so it's a question to spend our resources for this idea. Thirdly, **fheroes2** project is for Heroes of Might and Magic 2 only. Lastly, making such clone you should comply with all licensing issues.
